monitoring/src/check-page/common-functions.ts

```diff
@@ -57,11 +61,12 @@ export const clearLocalStorage = async (page: Page): Promise<void> => {
 const initialiseOptions = async (
 	isDebugMode: boolean,
 ): Promise<CustomPuppeteerOptions> => {
+	log_info(`chromium executable path: ${(process.env.IS_LOCAL == 'true') ? '/opt/homebrew/bin/chromium' : await chromium.executablePath()}`)
 	return {
 		headless: !isDebugMode,
-		args: isDebugMode ? ['--window-size=1920,1080'] : Chromium.args,
-		defaultViewport: Chromium.defaultViewport,
-		executablePath: await Chromium.executablePath,
+		args: isDebugMode ? ['--window-size=1920,1080'] : chromium.args,
+		defaultViewport: chromium.defaultViewport,
+		executablePath: (process.env.IS_LOCAL == 'true') ? '/opt/homebrew/bin/chromium' : await chromium.executablePath(),
```

Comment:
This might be a fix for the issue you're facing, but I'm not at all certain:
```suggestion
		executablePath: (process.env.IS_LOCAL == 'true') ? '/opt/homebrew/bin/chromium' : await chromium.executablePath('/opt/nodejs/node_modules/@sparticuz/chromium/bin'),
```
I found this from [this issue](https://github.com/Sparticuz/chromium/issues/41#issuecomment-1407545726)